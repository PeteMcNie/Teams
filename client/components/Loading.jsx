import React from 'React'
import { connect } from 'react-redux'

const Loading = (props) => {

}
function mapStateToProps (state) {
    return {
        
    }
}

export default connect(mapStateToProps)(Loading)
