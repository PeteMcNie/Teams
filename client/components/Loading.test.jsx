import React from 'react'

import { renderWithRedux } from '../testing/utils'

import '@testing-library/jest-dom'

import Loading from './Loading'

test('Loading component displays if loading is true', () => {
  const { asFragment } = renderWithRedux(<Loading />,
    { initialState: { loading: true } })
  expect(asFragment()).toMatchSnapshot()
})

test('Loading component does not display if loading is false', () => {
  const { asFragment } = renderWithRedux(<Loading />,
    { initialState: { loading: false } })
  expect(asFragment()).toMatchSnapshot()
})
