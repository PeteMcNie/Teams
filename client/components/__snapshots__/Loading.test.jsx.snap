// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component displays if loading is true 1`] = `
<DocumentFragment>
  <img
    class="loading"
    src="/images/loading.svg"
  />
</DocumentFragment>
`;

exports[`Loading component does not display if loading is false 1`] = `<DocumentFragment />`;
